src/ExaPF.jl

```diff
@@ -188,7 +188,7 @@ function polar!(Vm, Va, V, ::CUDADevice)
     Va .= CUDA.angle.(V)
 end
 
-function solve(pf::PowerSystem.PowerNetwork, npartitions=2, solver="default";
+function solve(pf::PowerSystem.PowerNetwork, x, u, p, npartitions=2, solver="default";
```

Comment:
I would maybe add proper type annotation for vector there: 
```julia
x::AbstractArray,
u::AbstractArray,
p::AbstractArray,
```

Reply:
Right!